IngressMaxFields.user.js

```diff
@@ -69,66 +69,79 @@ function wrapper() {
 		});
 		return c;
 	}
+	self.inBounds = function(portal) {
+		if (window.plugin.drawTools && window.plugin.drawTools.drawnItems.getLayers().length) {
+			return self.portalInDrawnItems(portal);
+		} else {
+			return self.portalInScreen(portal);
+		}
+	}
+	self.genStr = function genStr(p) {
+		var href = 'https://www.ingress.com/intel?ll=' + p._latlng.lat + ',' + p._latlng.lng + '&z=17&pll=' + p._latlng.lat + ',' + p._latlng.lng;
+		var str = p.options.data.title
+		var str = str.replace(/\"/g, "\\\"");
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Variable 'str' is already declared in line 81.](https://www.codacy.com/app/stefan-nygren/IITC-Ingress-Maxfields-Exporter/file/1581187182/issues/source?bid=2892513&fileBranchId=3023982#l82)